core/dbt/main.py

```diff
@@ -448,6 +449,21 @@ def _build_debug_subparser(subparsers, base_subparser):
     return sub
 
 
+def _build_manage_subparser(subparsers, base_subparser):
```

Comment:
This parameter definition and how we invoke the manage task will need to be moved to the new [params.py](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/cli/params.py) and click command([link](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/cli/main.py))